### PR TITLE
correcting naming and links in README

### PR DESCRIPTION
**Pull request by Ritikaa96**
Signed-off-by: Ritikaa96 <ritika@india.nec.com>

There is a correction needed here. 
>  See "scrips" in package.json.

Replacing 'scrips' with 'scripts' , Ava's link with correct ones. 

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))

### CODE REVIEW

**Comment by Ritikaa96** [comment]
@tomkerkhove @zroubalik PTAL.

**Review by tomkerkhove — APPROVED** [review]

**Comment by Ritikaa96** [comment]
Hi, @tomkerkhove 

> `setup.test.ts` deploys [`/deploy/KedaScaleController.yaml`](../deploy/KedaScaleController.yaml) to `keda` namespace in the cluster, and updates the image to `kedacore/keda:master`

This link with KedaScaleController.yaml is dead. can we skip the link and file name as it deploys keda with :

 
```
test.serial('Deploy Keda', t => {
  let result = sh.exec('(cd .. && make deploy)')
  if (result.code !== 0) {
    t.fail('error deploying keda. ' + result)
  }
  t.pass('Keda deployed successfully using make deploy command')
})
```

and there is no deploy folder anymore neither the kedaScaleController.yaml.

**Review by zroubalik — APPROVED** [review]
Good catch, thx

**Comment by zroubalik** [comment]
> Hi, @tomkerkhove
> 
> > `setup.test.ts` deploys [`/deploy/KedaScaleController.yaml`](../deploy/KedaScaleController.yaml) to `keda` namespace in the cluster, and updates the image to `kedacore/keda:master`
> 
> This link with KedaScaleController.yaml is dead. can we skip the link and file name as it deploys keda with :
> 
> ```
> test.serial('Deploy Keda', t => {
>   let result = sh.exec('(cd .. && make deploy)')
>   if (result.code !== 0) {
>     t.fail('error deploying keda. ' + result)
>   }
>   t.pass('Keda deployed successfully using make deploy command')
> })
> ```
> 
> and there is no deploy folder anymore neither the kedaScaleController.yaml.

Yes please!

**Comment by Ritikaa96** [comment]
> > Hi, @tomkerkhove
> > > `setup.test.ts` deploys [`/deploy/KedaScaleController.yaml`](../deploy/KedaScaleController.yaml) to `keda` namespace in the cluster, and updates the image to `kedacore/keda:master`
> > 
> > 
> > This link with KedaScaleController.yaml is dead. can we skip the link and file name as it deploys keda with :
> > ```
> > test.serial('Deploy Keda', t => {
> >   let result = sh.exec('(cd .. && make deploy)')
> >   if (result.code !== 0) {
> >     t.fail('error deploying keda. ' + result)
> >   }
> >   t.pass('Keda deployed successfully using make deploy command')
> > })
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > and there is no deploy folder anymore neither the kedaScaleController.yaml.
> 
> Yes please!

Done. PTAL @zroubalik

File: tests/README.md [hunk 1]
```diff
@@ -20,9 +20,9 @@ npx ava scalers/prometheus.test.ts
 ## E2E test setup
 
 The test script will run 3 phases:
-- **Setup**: this is done in [`setup.test.ts`](setup.test.ts). If you're adding any tests for KEDA install/setup process add it to this file.`setup.test.ts` deploys [`/deploy/KedaScaleController.yaml`](../deploy/KedaScaleController.yaml) to `keda` namespace in the cluster, and updates the image to `kedacore/keda:master`
+- **Setup**: this is done in [`setup.test.ts`](setup.test.ts). If you're adding any tests for KEDA install/setup process add it to this file.`setup.test.ts` deploys keda to `keda` namespace in the cluster, and updates the image to `kedacore/keda:master`.
```
Comment (zroubalik) [inline tests/README.md:23]:
```suggestion
- **Setup**: this is done in [`setup.test.ts`](setup.test.ts). If you're adding any tests for KEDA install/setup process add it to this file.`setup.test.ts` deploys KEDA to `keda` namespace in the cluster, and updates the image to `kedacore/keda:main`.
```

**Review by zroubalik — COMMENTED** [review]
Minor nit

File: tests/README.md (hunk 1 above)
Comment (Ritikaa96) [reply tests/README.md:23]: oh right. wonder how i skipped it. will edit it @zroubalik . thanks.

Comment (Ritikaa96) [reply tests/README.md]: Done. @zroubalik  PTAL

**Review by zroubalik — APPROVED** [review]
LGTM, thanks!

**Pull request merged**